### ISSUE TITLE
Goto implementation UI has a contrast issue?

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#212548]
I find this representation to lack contrast:

![image](https://github.com/microsoft/vscode/assets/900690/5b2c9a93-9202-48a8-be85-cedbdd8420d5)

I am not sure it meets our a18y requirements?

This is the "Find Implementations" peek UI.

**Comment by schwaber** [comment]
The root cause of this problem is described in detail in issue #211623, but unfortunately closed as duplicate, even it is dated Apr 29, 2024, without fixing the error.

**Comment by schwaber** [comment]
Congratulations.

A year ago, this and other issues were reported and have not been resolved to this day, and they are clearly no longer on the list of errors to be fixed.

You really have an **accessibility problem**, as you do not have access to dedicated software and test engineers who want to eliminate their own mistakes, but rather hide in the wake of other reports.

Setting "list.highlightForeground" to "#ffffff" fixes the color in the peekViewResult view, but the "quickOpen" view also uses this color definition and thus quickInput's highlightForeground is then shown in white and not as expected in "#2aaaff" (blue).

The **proposed and only solution** is to undo the changes in "#208108" and to check if the reported error "#207744" still exists in the "quickOpen view".

The issue "#207744" did not occur up to "Visual Studio Code 1.87.2" if ever, but the issue "#212548" described above only occurs after merging "#208108" into the release of "Visual Studio Code 1.88.0".

The only accepted probable "bug" is that the contrast between foreground and background colors is too low.
